README.md

```diff
@@ -12,23 +12,93 @@ Github CI
 - **kuka_kr_moveit_config** contains configuration files for KUKA KR robots necessary for planning with moveit.
 - **kuka_lbr_iisy_support** contains urdf, config and mesh files for KUKA iisy robots.
-- **kuka_lbr_iisy_moveit_config** contains configuration files for KUKA iisy robots necessary for planning with moveit.
+- **kuka_lbr_iisy_moveit_config** contains configuration files for KUKA LBR iisy robots necessary for planning with moveit.
 - **kuka_agilus_support** contains urdf, config and mesh files for KUKA Agilus robots, it is copied from [kuka_experimental](https://github.com/ros-industrial/kuka_experimental) and ported to ROS2.
-- **kuka_lbr_iiwa_support** contains urdf, config and mesh files for KUKA LBR iiwa 7 robots
-- **kuka_rsi_simulator** implements a basic RSI simulator. It is also copied from [kuka_experimental](https://github.com/ros-industrial/kuka_experimental) and is ported to ROS2. It implements a UDP socket, and receives the commands from an RSI interface and sends back the actual position of the robot (as it is a simulation, it is a simple closed loop, it sends what it received).
+- **kuka_lbr_iiwa_support** contains urdf, config and mesh files for KUKA LBR iiwa robots
+- **kuka_lbr_iiwa_moveit_config** contains configuration files for KUKA LBR iiwa robots necessary for planning with moveit.
 
-## Running the RSI simulator
+## Structure of the support packages
 
+All support packages consist of 4 folders:
+- config: contains joint limits, necessary for time parametrization of trajectories
+- launch: contains launch files to be able to visualize the robot models
+- meshes: contains collision and visual meshes for the robots
+- urdf: contains the xacro files describing the robots, including ros2_control integration (with fake hardware argument)
+
+### Xacro files
+ Each robot has two specific xacro files: a macro ({*robot_name*}_macro.xacro) and another file instantiating this macro ({*robot_name*}.urdf.xacro). Additionally there is a xacro providing ros2_control integration, including the name and type of the hardware interface, hardware parameters and the supported state and command interfaces.
+ Additionally a transmission xacro is provided for gazebo support, but the mechanicalReduction parameters contained within are not valid, only placeholders.
+
+ The macro files contain the links and joints of the main serial chain, including transformations, rotation axes, inertial properties, joint position, velocity and effort limits and the location of the mesh files.
+
+ The macro file follows the ROS-Industrial conventions:
+ - link names are "link_{*i*}"
+ - joint names are "joint_{*i*}"
+ - all link and joint names have a {prefix} argument
+ - base frame: equivalent to the base frame defined by the industrial controller ($ROBROOT)
+ - flange frame: attachment point for EEF models
+ - tool0 frame: all-zeros tool frame
+
+ The frames in the xacros follow the Denavit–Hartenberg conventions of Khalil-Dombre.
```

Comment:
This is a rather significant divergence from "all" other ROS-Industrial `xacro:macro`s, and also different from many other, non-industrial robot URDFs.

While it seems to make sense in your case, I would perhaps suggest to describe this in a separate section.

I'd also suggest to clarify frames `base`, `flange` and `tool0` do follow ROS-Industrial conventions. `flange` specifically should follow REP-103 wrt chirality and orientation.

Reply:
I have just realized, that currently we do not follow this convention for the flange, I will also fix that in this PR.
@gavanderhoorn  I also have two questions regarding this: there are a few KUKA robots, where the all-zeros joint configuration is not valid (eg. KR 210 R2700-2), what should be done for these robots? Should REP-103 be valid for the all-zeros configuration, although it cannot be reached? 

As far as I understood, this convention is designed to make attaching end effectors easier, does that also mean that the x+ axis should be always perpendicular to the surface of the flange? The LBR iiwa's flange points upwards, while the KR robots' point forward, which means that attaching an EEF would be different, if REP-103 is fulfilled in the null position.